cli/src/lib.rs

```diff
@@ -115,7 +116,7 @@ pub struct NodeBuilder {
 impl NodeBuilder {
     /// Creates new builder.
     pub fn new() -> Self {
-        Self::default()
+        Self::default().with_service(Supervisor)
```

Comment:
Are you sure that Supervisor will start with the default instance?

Reply:
Good catch.